server/routes/dashboardCustomScripts.tsx

```diff
@@ -21,13 +21,13 @@ app.get('/dash/scripts', async (req, res, next) => {
 		}
 
 		const customScripts = await getCustomScriptsForCommunity(communityData.id);
-
+		const communityId = { id: communityData.id };
```

Comment:
As mentioned above the `usePageContext()` hook (which, n.b., is reading from the `initialData` we pass in here) makes it unnecessary to pass this down here.